Automattic/woocommerce-payments
# Record a tracks event when a merchant is redirected to a Stripe Capital loan offer

Fixes #9651

#### Changes proposed in this Pull Request

This PR adds a new tracks event **`wcadmin_wcpay_capital_view_offer_redirect`** when a merchant is redirected to the Stripe Capital offer page, after clicking the View Offer/Details link in the offer email.

Adding this event helps us to track how well Stripe Capital flows work (converts to loans) and if merchants are dropping off anywhere in the flow.


#### Testing instructions

- Navigate to the admin path `/wp-admin/admin.php?wcpay-loan-offer`
- View the tracks event live view for [`wcadmin_wcpay_capital_view_offer_redirect`](https://mc.a8c.com/tracks/live/?eventname=wcadmin_wcpay_capital_view_offer_redirect&user=&useragent=) and wait ~10mins – ensure your account shows up in this list
- If you would like to test the entire Capital Loans flow, see Helix PdjTHR-31T-p2 for testing tips, **but this is optional, since this tracks event is isolated to the redirect whether an offer is available or not.**

-------------------


**Post merge**



> [!IMPORTANT]  
> Don't forget post-merge tasks!

- [x] Register `wcadmin_wcpay_capital_view_offer_redirect` tracks event
	- See registration PR https://github.com/Automattic/tracks-events-registration/pull/2647
- [x] Add the new tracks event to the Helix Capital Cheatsheet: PdjTHR-31T-p2
- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : __QA Testing Not Applicable__
